app/gitcoinbot/gitcoinbotActions.py

```diff
@@ -0,0 +1,193 @@
+'''
+Methods for interacting with the Github API
+'''
+from django.conf import settings
+
+import datetime
+import json
+import jwt
+import requests
+import re
+
+_auth = (settings.GITHUB_API_USER, settings.GITHUB_API_TOKEN)
+GITCOINBOT_APP_ID = settings.GITCOINBOT_APP_ID
+SECRET_KEYSTRING = settings.SECRET_KEYSTRING
+
+headers = {
+    'Accept': 'application/vnd.github.squirrel-girl-preview'
+}
+
+def help_text():
+    # TODO @owocki Add test bounties to dev environment to add claim/approve functionality
+    # help_text_response = "I am @{}, a bot that facilitates gitcoin bounties.\n".format(settings.GITHUB_API_USER) + \
+    #     "\n" +\
+    #     "<hr>" +\
+    #     "Here are the commands I understand:\n" +\
+    #     "\n" +\
+    #     " * `bounty <amount> ETH` -- receive link to gitcoin.co form to create bounty.\n" +\
+    #     " * `claim` -- receive link to gitcoin.co to claim bounty.\n" +\
+    #     " * `approve` -- receive link to gitcoin.co to approve bounty.\n" +\
+    #     " * `tip <user> <amount> ETH` -- receive link to complete tippping another github user *<amount>* ETH.\n" +\
+    #     " * `help` -- displays a help menu\n" +\
+    #     "\n" +\
+    #     "<br>" +\
+    #     "Learn more at: [https://gitcoin.co](https://gitcoin.co)\n" +\
+    #     ":zap::heart:, {}\n".format("@" + settings.GITHUB_API_USER)
+
+    help_text_response = "I am @{}, a bot that facilitates gitcoin bounties.\n".format(settings.GITHUB_API_USER) + \
+    "\n" +\
+    "<hr>" +\
+    "Here are the commands I understand:\n" +\
+    "\n" +\
+    " * `bounty <amount> ETH` -- receive link to gitcoin.co form to create bounty.\n" +\
+    " * `tip <user> <amount> ETH` -- receive link to complete tippping another github user *<amount>* ETH.\n" +\
+    " * `help` -- displays a help menu\n" +\
+    "\n" +\
+    "<br>" +\
+    "Learn more at: [https://gitcoin.co](https://gitcoin.co)\n" +\
+    ":zap::heart:, {}\n".format("@" + settings.GITHUB_API_USER)
+    return help_text_response
+
+def new_bounty_text(owner, repo, issue_id, comment_text):
+    issue_link = "https://github.com/{}/{}/issues/{}".format(owner,
+                                                            repo,
+                                                            issue_id )
+    bounty_amount = parse_comment_amount(comment_text)
+    # prod
+    # bounty_link = "https://gitcoin.co/funding/new?source={}&amount={}".format(
+    #     issue_link, bounty_amount
+    # )
+    #dev
+    bounty_link = "http://localhost:8000/funding/new?source={}&amount={}".format(
```

Comment:
This should be dynamic. You can use `settings.BASE_URL` coupled with `reverse` to fetch the `/funding/new` and use literal string interpolation (f-strings) when possible versus `.format()`